code/modules/client/preferences_savefile.dm

```diff
@@ -462,7 +490,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 			pref_species = new newtype
 
 
-		scars_index = rand(1,5)
+	scars_index = rand(1,5) // WHY
```

Comment:
i'm not sure this is necessary, true *and* kind

Reply:
salt because this is supposed to load without the species check